webpack_config/webpack.base.js

```diff
@@ -20,19 +20,9 @@ module.exports = {
   },
   resolve: {
     extensions: ['.js', '.jsx', '.css', '.json', '.scss', '.less'],
-    alias: {
-      actions: `${config.srcPath}/actions/`,
-      api: `${config.srcPath}/api/`,
-      reducers: `${config.srcPath}/reducers/`,
-      components: `${config.srcPath}/components/`,
-      containers: `${config.srcPath}/containers/`,
-      styles: `${config.srcPath}/styles/`,
-      less_vars: `${config.srcPath}/styles/etherwallet-variables.less`
-    },
-    // FIXME why aliases then?
```

Comment:
Why indeed? Removing them seemed to work fine, and I was in this file figuring out SASS resolves anyway, so I nix'd em.